cmake/modules/SwiftHandleGybSources.cmake

```diff
@@ -113,10 +121,20 @@ function(handle_gyb_sources dependency_out_var_name sources_var_name arch)
     if(src STREQUAL src_sans_gyb)
       list(APPEND de_gybbed_sources "${src}")
     else()
+
+      # On Windows (using Visual Studio), the generated project files assume that the
+      # generated GYB files will be in the source, not binary directory.
+      # TODO: when we use Ninja as a generator, this may be unecessary.
+      IF (WIN32)
```

Comment:
Lowercase `if`, no space after it.  Doesnt this mean that you are no longer doing an out-of-source compilation?